pkg/controller/operatingsystemconfig/actuator.go

```diff
@@ -98,9 +98,11 @@ if [ ! -s /etc/containerd/config.toml ]; then
 fi
 mkdir -p /etc/systemd/system/containerd.service.d
 cat <<EOF > /etc/systemd/system/containerd.service.d/11-exec_config.conf
+# remove this file once flatcar versions that use torcx are no longer supported
```

Comment:
Minor suggestion so that it is easier to look for TODOs and we don't forget them.
```suggestion
# TODO(MichaelEischer): remove this file once flatcar versions that use torcx are no longer supported
```

You could also apply this to similar comments in the PR

Reply:
I've updated the comments accordingly and included the flatcar version from which on only systemd-sysext is supported.